odoo/addons/base/migrations/10.0.1.3/noupdate_changes.xml

```diff
@@ -9,6 +9,17 @@
     <record id="tj" model="res.country">
         <field name="currency_id" ref="TJS"/>
     </record>
+    <record id="HUF" model="res.currency">
+        <field name="rounding">1.00</field>
+    </record>
+    <record id="user_root" model="res.users">
```

Comment:
Better to comment out this change as people might change previously the signature at their taste.

Reply:
True!

Reply:
Done!